__tests__/tests/components/StateBadge.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 
-import { closed as closedIssue, open as openIssue } from '../../data/issue';
+import { closed as closedIssue, open as openIssue } from '../../data/api/issue';
 import {
   closed as closedPr,
   open as openPr,
   merged as mergedPr,
-} from '../../data/pull-request';
+} from '../../data/api/pull-request';
```

Comment:
Also we're using relative imports here.

Reply:
These are just tests I touched when I moved the test data files, my discipline tells me update all tests in a different PR but I'll do it here.

Reply:
@alejandronanez all references updated to use alias instead of absolute paths.